contract/contracts/AstarBase.sol

```diff
@@ -43,13 +44,6 @@ contract AstarBase is Ownable {
         registeredCnt.increment();
     }
 
-    /// @notice unRegister senders' address
-    function unRegister() public {
-        require(!paused, "The contract is paused");
-
-        unRegisterExecute(msg.sender);
-    }
-
     /// @notice unRegister any address by contract owner
     /// @param evmAddress, EVM address used for registration
     function sudoUnRegister(address evmAddress) public onlyOwner {
```

Comment:
I'd recommend making this callable by anyone, not just sudo.

Also, make the call payable - it doesn't have to be anything extravagant - e.g. 50 ASTR would probably be fine.

Reply:
I just deleted unregister for everyone